### ISSUE TITLE
Is LevelDB a good fit for a RRD ?

### ISSUE THREAD

**Issue by jpillora** [Level/community#55]
Thinking about implementing a backend for https://github.com/etsy/statsd/ using levelup. Are there any insights anyone can provide me with :) ?

**Comment by ralphtheninja** [comment]
Fair enough. I realized I was jumping the gun on closing a lot of issues. Changed my mind and re-opened and moved to `community` repo instead, because a lot of the discussions were really interesting to keep around.
